src/main/docs/guide/views/templates/jstachio.adoc

```diff
@@ -0,0 +1,43 @@
+Micronaut Views JStachio includes api:views.jstachio.JStachioViewsRenderer[] which uses
+https://github.com/jstachio/jstachio[JStachio] A type-safe compiled Mustache based template engine.
+
+Add the `micronaut-views-jstachio` dependency to your classpath.
+
+dependency:micronaut-views-jstachio[groupId="io.micronaut.views"]
```

Comment:
Does the user need to add the jstachio's annotation processor to the annotation processor classpath?

Reply:
Yes further down in the doc it says to go to the JStachio documentation for that. The reason being is that you do not need to add the apt to the annotation processor path for Maven.

I can certainly copy the information from the JStachio documentation to the Micronaut adoc... I just was lazy and didn't want to have to keep it updated.

Reply:
Doc on that being here: https://jstach.io/doc/jstachio/1.2.1/apidocs/#installation

Reply:
the annotation processor is necessary for Gradle as well is not it? You add it in the test classpath.

Reply:
I think we should have in the Micronaut Views documentation every dependency necessary to get going.

Reply:
Yes I what I meant more or less about the doc link is that annotation processor setup is complicated and I am continuing to add notes on things like IDE as well.

But yeah we just want people to get started quickly.

What I will do is put the default annotation processor setup which is using `annotationProcessorPaths` for Maven and `annotationProcessor` for Gradle in the doc and then a link back to JStachio official documentation for extended annotation processing config and support.